lib/util/include/hse_util/hse_err.h

```diff
@@ -64,9 +64,9 @@
 static char _hse_merr_file[] _merr_attributes = __BASE_FILE__;
 
 extern char hse_merr_base[];
-extern char hse_merr_bug0[];
-extern char hse_merr_bug1[];
-extern char hse_merr_bug2[];
+extern char invalid_file_bug0[];
+extern char invalid_file_bug1[];
+extern char invalid_file_bug2[];
```

Comment:
I'd prefer you keep the original symbol names so that when I look for "hse_merr" in objdump these wont be missed.